### ISSUE TITLE
Table is not alignet in latex with table directive

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#506]
In latex a table with a csv-table directive is allignet at the right side of the end of the previous paragraph instead at the left of the page.

The following diff corrects it (for me):

```
#!python

diff -r 91d25c39b728 sphinx/writers/latex.py
--- a/sphinx/writers/latex.py   Sat Aug 14 19:48:42 2010 +0200
+++ b/sphinx/writers/latex.py   Tue Aug 17 22:34:50 2010 +0200
@@ -613,7 +613,7 @@
             self.table.longtable = True
         self.body = self._body
         if not self.table.longtable and self.table.caption is not None:
-            self.body.append(u'\n\\begin{threeparttable}\n'
+            self.body.append(u'\n\n\\begin{threeparttable}\n'
                              u'\\caption{&#37;s}\n' &#37; self.table.caption)
         if self.table.longtable:
             self.body.append('\n\\begin{longtable}')
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/506
- Originally reported by: Anonymous
- Originally created at: 2010-08-17T23:02:10.213

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-03 22:36:21+00:00_

I think this should have been resolved with <<cset 0cda61e88d903f5c675dc0acb193ca1e861ad751>>.

**Comment by shimizukawa** [comment]
_From Benoît Allard on 2011-08-01 13:06:09+00:00_

I do have that trouble with the following code:

{{{
#!rst
This is some text ending with:
.. _table:
.. table::

   ...
}}}

Which generates the following:

{{{
#!latex
This is some text ending with:
\phantomsection\label{doc:table}
\begin{threeparttable}
...
}}}

And my table get aligned at the right side of the end of the previous paragraph instead at the left of the page.

I do confirm that to reproduce the trouble, I need the following three components:
- text ending with `:`
- silent label
- table

**edit:** And I do also confirm that the included patch above solves it for me.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-23 06:04:53+00:00_

Fix #506: block table alignment problem.

→ <<cset 806cadaa01f4e5f5fcc01643a1eaced4061106bd>>
